http/src/main/resources/leo.conf

```diff
@@ -156,6 +156,17 @@ auth {
 }
 
 azure {
+  hosting-mode-config{
```

Comment:
So these env variables will come from your own leonardo helmchart?

Reply:
Yep, however, I could use some help to confirm my assumptions. A value in azure.hosting-mode-config is required. For GCP, I want to set the default to false to make it transparent. I did this by explicitly setting a default value in the config class (for clarity about the intent), as well as in the reference.conf. Is this sufficient, or do I need to update a deployment helm chart to include this value and ensure this change is transparent?

Reply:
I think it should be sufficient, I would not want to add azure deployment specific values to the leonardo helm chart that is designed to be deployed on GCP only 👍